modules/apps/frontend-view-state/app.bnd

```diff
@@ -1,6 +1,6 @@
 Liferay-Releng-App-Description: %liferay-releng-app-description
 Liferay-Releng-App-Title: ${liferay.releng.app.title.prefix} Frontend View State
-Liferay-Releng-Bundle: true
+Liferay-Releng-Bundle: false
```

Comment:
@izaera SF enforce `Liferay-Releng-Bundle` to `true`.
@jpince can you help to check what SF should fix?

/cc @hhuijser

Reply:
:scream: Sorry about that. I couldn't imagine that changing only that line could trigger that SF failure :-(